hack/lib/build/environment.sh

```diff
@@ -173,12 +175,21 @@ function os::build::environment::run() {
   if [[ -z "${volume}" ]]; then
     volume="origin-build-$( git rev-parse "${commit}" )"
   fi
+  volume="$( echo "${volume}" | tr '[:upper:]' '[:lower:]' )"
+  export OS_BUILD_ENV_VOLUME="${volume}"
+
+  if [[ -n "${OS_BUILD_NEW_VOLUME:-}" ]]; then
+    if docker volume inspect "${volume}" >/dev/null 2>&1; then
```

Comment:
I strongly believe that is an antipattern that promotes cargo-culting by obfuscating what is actually going on by using arcane defaults that hide file descriptor redirection.